spec/cordova/platforms/platforms.spec.js

```diff
@@ -62,7 +62,7 @@ describe('platforms/platforms', () => {
             expect(platformApi).toBeDefined();
             expect(platformApi.platform).toEqual('windows');
             expect(events.emit.calls.count()).toEqual(1);
-            expect(events.emit.calls.argsFor(0)[1]).toEqual('PlatformApi successfully found for platform windows');
+            expect(events.emit.calls.argsFor(0)[1]).toMatch('PlatformApi successfully found');
```

Comment:
```suggestion
            expect(events.emit.calls.argsFor(0)[1]).toMatch('Platform API successfully found in:');
```
If you accept the emit change below, I went ahead and updated this as well..